newsletters/2023-03-newsletter.md

```diff
@@ -36,30 +36,33 @@ Hello! All Vipps customers that have API access now receive this technical
 newsletter.
 
 The newsletter is sent 5-10 times a year and contains important updates and
-information about our APIs: https://vippsas.github.io/vipps-developer-docs/docs/APIs
+information about our [APIs](https://vippsas.github.io/vipps-developer-docs/docs/APIs).
 
-Newsletter archive: https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/newsletters
+See: [Newsletter archive](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/newsletters).
 
 (You can unsubscribe if you want - see the link at the bottom)
 
 ## Improved shipping options in Vipps Checkout
 
-To increase conversion in your webshop and create a great online customer experience, giving your customers an easy way of choosing their preferred shipping method is essential. We've done the heavy lifting, together with the shipping providers, so you can focus on other things.
+To increase conversion in your webshop and create a great online customer experience, it is essential that you provide your customers with an easy way of choosing their preferred shipping method. We've done the heavy lifting, together with the shipping providers, so you can focus on other things.
```

Comment:
I think we should delete:  "We've done the heavy lifting, together with the shipping providers, so you can focus on other things."
because our goal is to reduce documentation and that sentence doesn't really provide anything useful. It also sounds like we are proud of ourselves and is a bit annoying.

Reply:
```suggestion
To increase conversion in your webshop and create a great online customer experience, it is essential that you provide your customers with an easy way of choosing their preferred shipping method.
```